app/services/discourse_reactions/reaction_manager.rb

```diff
@@ -22,6 +22,7 @@ def toggle!
     private
 
     def toggle_like
+      PostAction.limit_action!(@user, @post, post_action_like_type)
```

Comment:
This will need a test

Reply:
I think this has no need here as its already called with `PostActionCreator`